packages/tocco-theme/src/ToccoTheme/theme.scss

```diff
@@ -0,0 +1,32 @@
+/* tocco theme for react components
+ * CSS class .tocco-ui-theme is used to scope styles.
+ * CSS class .tocco-ui-theme is used to increase specificity compared to standard vendors implementation. Hence Tocco's Bootstrap has a higher specificity than Bootstrap.
```

Comment:
please add a max line length of 120. Then its consistent with js-linting and we dont have a dependency on any IDE